src/label/creator.py

```diff
@@ -204,7 +215,7 @@ def create_box_label(member_number, name):
     return label_image
 
 
-def create_chemical_storage_label(member_id, name):
+def create_fire_box_storage_label(member_id, name):
 
```

Comment:
Add a logo/title like "Fire box item", as for [temporary storage labels](https://github.com/makerspace/memberbooth/blob/16ba30c2b02fb4d8fd551c9f7267e9cabaf91eb8/src/label/creator.py#L65-L66)

Reply:
I agree.